site/content/in-dev/unreleased/metastores.md

```diff
@@ -29,14 +29,45 @@ deprecated EclipseLink persistence backends.
 This implementation leverages Quarkus for datasource management and supports configuration through
 environment variables or JVM -D flags at startup. For more information, refer to the [Quarkus configuration reference](https://quarkus.io/guides/config-reference#env-file).
 
+We have 2 options for configuring the persistence backend:
 
+### 1. Relational JDBC metastore with username and password
+
+using environment variables:
 ```
 POLARIS_PERSISTENCE_TYPE=relational-jdbc
 
 QUARKUS_DATASOURCE_USERNAME=<your-username>
 QUARKUS_DATASOURCE_PASSWORD=<your-password>
 QUARKUS_DATASOURCE_JDBC_URL=<jdbc-url-of-postgres>
 ```
+using properties file:
+
+```
+polaris.persistence.type=relational-jdbc
+quarkus.datasource.jdbc.username=<your-username>
+quarkus.datasource.jdbc.password=<your-password>
+quarkus.datasource.jdbc.jdbc-url=<jdbc-url-of-postgres>
+```
```

Comment:
please check this feedback from @dimas-b 
https://github.com/apache/polaris/pull/1515/files#r2074243975

IMHO we should be consistent, as far as application.propeties is concerned we reference the quarkus conf page above so that audience can choose what ever way they prefer.

Reply:
I guess that's right 😅 I think the PR is good to merge :)